services/feed-manager-service/feed-manager-controller/src/main/java/com/thinkbiganalytics/feedmgr/service/feed/FeedModelTransform.java

```diff
@@ -388,6 +405,21 @@ public FeedSummary domainToFeedSummary(@Nonnull final Feed feedManagerFeed) {
     public List<FeedSummary> domainToFeedSummary(@Nonnull final Collection<? extends Feed> domain) {
         return domain.stream().map(this::domainToFeedSummary).filter(feedSummary -> feedSummary != null).collect(Collectors.toList());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with method reference 'Objects::nonNull'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)